File: Data Structures & Algorithms/sliding-window-maximum/submission-0.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:

        res = []

        
        for i in range(0,len(nums)-k+1):

            Max_elements = []

            for j in range(i,i+k):

                Max_elements.append(nums[j])
            
            res.append(max(Max_elements))
        
        return res



        

File: Data Structures & Algorithms/maximum-product-subarray/submission-0.java
class Solution {
    public int maxProduct(int[] nums) {
        int n=nums.length;

        int max_product=Integer.MIN_VALUE;


        for(int i=0;i<n;i++)
        {
            int res=nums[i];
            max_product=Math.max(max_product,res);
            for(int j=i+1;j<n;j++)
            {
                res=res*nums[j];
                max_product=Math.max(max_product,res);
            }
        }
        return max_product;


    }
}
